formula-android/src/main/java/com/instacart/formula/android/internal/CompositeBinding.kt

```diff
@@ -72,10 +72,7 @@ internal class CompositeBinding<ParentComponent, ScopedComponent>(
     override fun types(): Set<Class<*>> = types
 
     override fun binds(key: Any): Boolean {
-        bindings.forEachIndices {
-            if (it.binds(key)) return true
-        }
-        return false
+        return types.contains(key.javaClass)
```

Comment:
We should have all `types` in a set, so no need to iterate over `bindings` manually.